acceptance.bats

```diff
@@ -59,6 +59,11 @@
   [[ "$output" =~ "data.kubernetes.is_service" ]]
 }
 
+@test "Test command with json output and trace" {
+  run ./conftest test -p examples/kubernetes/policy examples/kubernetes/deployment.yaml -o json --trace
+  [ "$status" -eq 1 ]
```

Comment:
Your acceptance test only checks that there are failures and that the exit code is 1, not that traces are enabled. We would need to add something here that reads the output and looks for a trace.